inst/app/server.R

```diff
@@ -0,0 +1,212 @@
+server <- function(input, output, session) {
+  
+  # load(here("icd10_usage.rda"))
+  # load(here("snomed_usage.rda"))
+  
+  icd10_usage <- clinicalActivity::icd10_usage
+  snomed_usage <- clinicalActivity::snomed_usage
```

Comment:
I had a quick look and think we need to update this:

```suggestion 
  icd10_usage <- codeusage::icd10_usage
  snomed_usage <- codeusage::snomed_usage
```

Reply:
Thank you for catching this.